### ISSUE TITLE
Question about the Table 1. CAM evaluation.

### ISSUE THREAD

**Issue by Shield9999** [linyq2117/CLIP-ES#4]
First of all, thanks for the nice work! I have a question about your code. 
I hope the Table 1, seed and dCRF means respectively the initial+CAA and initial+CAA+dCRF CAM. 
So when I try to run the code eval_cam.py and eval_cam_with_crf.py, the performance results were respectively 65.96%, 68.76%.
It has lot of gap between the paper. However, the final performance with that CAM was similar to the paper's result.

Thank you for reading!

**Comment by linyq2117** [comment]
Thanks for your interest in our work. 

It seems that you evaluate the performance of CAM on the train_aug set (10582 images in total). However, the results in Table 1 are based on the train set (1464 images in total). You can change the `--split_file` to reproduce our results in Table 1. We have also changed it in `README.md` to avoid confusion.

**Comment by Shield9999** [comment]
Oh that's the reason! Sorry for my mistake, it was train_aug file. Thanks for the comment.

**Comment by henry189** [comment]
How to visualize refine cam?

**Comment by linyq2117** [comment]
> How to visualize refine cam?

To visualize cam, you can use `from pytorch_grad_cam.utils.image import show_cam_on_image` following instructions in [pytorch_grad_cam](https://github.com/jacobgil/pytorch-grad-cam/tree/61e9babae8600351b02b6e90864e4807f44f2d4a).
